hikari/errors.py

```diff
@@ -227,27 +214,31 @@ class HTTPResponseError(HTTPError):
     raw_body: typing.Any = attr.ib()
     """The response body."""
 
-    reason: typing.Optional[str] = attr.ib(default=None)
-    """The error reason. If `builtins.None`, will generate one automatically."""
-
-    message: str = attr.ib(default=_message_default_factory, init=False)
+    # We want to make it optional here, so ignore mypy complaining
+    message: typing.Optional[str] = attr.ib(default=None)  # type: ignore[assignment]
     """The error message."""
 
-    def __str__(self) -> str:
-        try:
-            raw_body = self.raw_body.decode("utf-8")
-        except (AttributeError, UnicodeDecodeError):
-            raw_body = str(self.raw_body)
-
-        chomped = len(raw_body) > 200
+    code: typing.Union[int] = attr.ib(default=0)
```

Comment:
union int?

Reply:
Oh, my bad, I used to have another thing there, will fix :sweat_smile: